source/api/presentation/3.0/index.md

```diff
@@ -1179,17 +1179,19 @@ Additional features of the [Web Annotation][webanno] data model _MAY_ also be us
 
 ###  5.6. Range
 
-Ranges are used to describe additional structure within an object, such as newspaper articles that span pages or chapters within a book. Ranges can include Canvases, parts of Canvases, or other Ranges, creating a nested tree structure such as a table of contents.
+Ranges are used to represent structure within an object beyond the overall Sequence, such as newspaper sections or articles, chapters within a book, or movements within a piece of music. Ranges can include Canvases, parts of Canvases, or other Ranges, creating a tree structure like a table of contents.
 
 The intent of adding a Range to the Manifest is to allow the client to display a hierarchical navigation interface to enable the user to quickly move through the object's content. Clients _SHOULD_ present only Ranges with the `label` property and without the "no-nav" `renderingHint` to the user. Clients _SHOULD NOT_ render Canvas labels as part of the navigation, and a Range that wraps the Canvas _MUST_ be created if this is the desired presentation.
 
-Ranges _MUST_ have URIs and they _SHOULD_ be http(s) URIs.  Ranges are embedded or referenced within the Manifest in a `structures` property. Ranges then embed or reference other Ranges, Canvases or parts of Canvases in the `items` field.  Each entry in the `items` field _MUST_ be a JSON object, and it _MUST_ have the `id` and `type` properties.
+Ranges _MUST_ have URIs and they _SHOULD_ be http(s) URIs.  Ranges are embedded or referenced within the Manifest in a `structures` property. Ranges embed or reference other Ranges, Canvases, or parts of Canvases in the `items` property.  Each entry in the `items` property _MUST_ be a JSON object, and it _MUST_ have the `id` and `type` properties.
 
-All of the Canvases or parts that should be considered as being part of a Range _MUST_ be included within the Range's members, or a descendant Range's members.
+All of the Canvases or parts that should be considered as being part of a Range _MUST_ be included within the Range's `items` list, or a descendant Range's `items`. 
 
-The Canvases and parts of Canvases may or may not be contiguous or in the same order as any Sequence.  Examples include newspaper articles that are continued in different sections, or simply a chapter that starts half way through a page. Parts of Canvases _MUST_ be rectangular and are described using the `xywh=` fragment approach.
+Parts of Canvases are still Canvases and have a `type` of "Canvas". Spatial parts of Canvases _MUST_ be rectangular and are described by appending an `xywh=` fragment to the end of the Canvas's URI. Temporal parts of Canvases _MUST_ be described by appending a `t=` fragment to the end of the Canvas's URI. Spatial and temporal fragments _MAY_ be combined, using an `&` character between them. 
```

Comment:
Should we say that spatial and temporal _SHOULD_ appear in a particular order such as  `#xywh=...&t=...` or `#t=...&xywh=...` (and _NOT_ the other) in order to have a canonical URI? In https://www.w3.org/TR/media-frags/#fragment-dimensions it says that the dimensions are independent but the [client guidelines](https://www.w3.org/TR/media-frags/#media-fragment-clients) say that clients are expected to do temporal first, then spatial at the codec level. That would suggest temporal first syntax might be the best option if we specify an order: `#t=...&xywh=...`

Reply:
If we are further constraining the fragments do we want to prohibit references to a non-existent dimension on the Canvas?  https://www.w3.org/TR/media-frags/#error-media-general says "If the user agent knows the mime type, it is able to detect non-existent dimensions and SHOULD ignore them."

Reply:
Given #1320, I think the definition of parts of Canvases should actually go under the Canvas definition as it will be used in multiple places.  Then Ranges, Annotations, `start` and anywhere else (that might be it?) can reference the Canvas section.  Thoughts?